scripts/zones/Ifrits_Cauldron/mobs/Bomb_Queen.lua

```diff
@@ -19,25 +19,15 @@ end
 function onMobFight(mob,target)
     local battleTime = mob:getBattleTime()
 
-    if battleTime > 20 and battleTime % 25 == 0 then
-        local mobId = mob:getID()
-        local pets = {}
-
-        for i = mobId + 1, mobId + 5 do
-            if not GetMobByID(i):isSpawned() then
-                table.insert(pets, i)
-            end
-        end
-
-        if #pets > 0 then
-            local petId = pets[math.random(#pets)]
-            local pet = GetMobByID(petId)
-            pet:setSpawn(mob:getXPos() - 0.5, mob:getYPos(), mob:getZPos(), mob:getRotPos())
-            pet:spawn()
-            pet:updateEnmity(target)
-        end
+    if battleTime - 5 % 20 == 0 then -- every 20 seconds after 5 seconds, source: https://www.youtube.com/watch?v=AVsEbYjSAHM
```

Comment:
I can't find specifics for LUA, but in most languages modulo occurs before subtraction in order of operations, so this would need some parentheses.

Reply:
Correct. Modulo is the "same" precedence as multiplication/division.

Reply:
fixed, thanks